model/src/main/proto/question.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package model;
+
+import "exploration.proto";
+
+option java_package = "org.oppia.app.model";
+option java_multiple_files = true;
+
+// Structure for a single question.
+message Question {
+  string question_id = 1;
+  State question_state = 2;
```

Comment:
Out of curiosity, how does routing work for questions? Presumably, the ``Outcome`` you match to an answer won't have a destination state name. Is this what the ``labelled_as_correct`` should be used for? Can you also route to an exploration from a question via the refresher exploration ID?

Reply:
Yes it doesnt have a destination state, it only uses labelled_as_correct, and it moves to the next question if the labelled_as_correct value is true (https://github.com/oppia/oppia/blob/40cb6adca9101f46209cecd4c83bd26a4f112bf9/core/templates/dev/head/pages/exploration-player-page/services/question-player-engine.service.ts#L206)

Reply:
Perfect, then I think I modeled this correctly downstream. Thanks!